### ISSUE TITLE
[UX] Layout listing: Do a better job at indicating that the paths in the table headings are actually paths

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4869]
This was brought up during the Dec. 31, 2020 weekly UX meeting by @philsward at ~https://youtu.be/Fc-oQ55LQi0?t=2058 (~34min into the call)

> ...the layout groups ...what are the thoughts on adding a forward-slash to the beginning of the path? ...I think that this was a big disconnect for me. ...I completely overlooked that the path was the header of the group, because when I saw "home", it's just like: oh, that's the header for "Home" ...sure, yeah, it's "Home" ...it says it's the default home here or whatever ...I didn't even realize until Tim and I started talking ...oh no, that's the _path_ that that layout is handling!

![Screen Shot 2021-01-11 at 5 15 31 pm](https://user-images.githubusercontent.com/2423362/104199916-b5c9d980-5430-11eb-9d93-3ab889e834eb.png)

> ...is there a reason for not having a forward-slash that I'm not seeing?

...@stpaultim said that he thought that that was interesting, @jenlampton nodded "no" (as in there is no reason to not do it), and went on to note that we could even go one step further, and add the whole URL of the site - not just the path ...and @cellear seemed to really like that idea!

**Comment by klonos** [comment]
...here's how that would look like:

![image](https://user-images.githubusercontent.com/2423362/104406492-1ad41b00-5568-11eb-9fbe-3781252e5950.png)

Does it improve things?

**Comment by klonos** [comment]
...or this perhaps:

![image](https://user-images.githubusercontent.com/2423362/104406953-3b50a500-5569-11eb-8ca8-14cadfce19f6.png)

**Comment by stpaultim** [comment]
I like the idea. I think that maybe the second option is sufficient and less cluttered.

**Comment by olafgrabienski** [comment]
> I like the idea. I think that maybe the second option is sufficient and less cluttered.

Yes, the second option looks much better. And having the prefix `Path:` we don't even need the starting slash (IIRC, usually we also don't use it otherwise).

Question: Why "Path and Layouts" (plural) in the 2nd screenshot?

**Comment by klonos** [comment]
> Question: Why "Path and Layouts" (plural) in the 2nd screenshot?

Because I realised that each path entry might have either one or multiple layouts. For example:

![image](https://user-images.githubusercontent.com/2423362/104430762-c80e5980-558f-11eb-8b92-9517f36d3e3b.png)
